src/sentry/static/sentry/app/views/discover/discover.tsx

```diff
@@ -405,111 +405,93 @@ export default class Discover extends React.Component<Props, State> {
       utc,
     } = this.props;
 
-    const currentQuery = queryBuilder.getInternal();
-
     const shouldDisplayResult = resultManager.shouldDisplayResult();
 
-    const start =
-      (currentQuery.start &&
-        moment(currentQuery.start)
-          .local()
-          .toDate()) ||
-      currentQuery.start;
-    const end =
-      (currentQuery.end &&
-        moment(currentQuery.end)
-          .local()
-          .toDate()) ||
-      currentQuery.end;
-
     return (
-      <DiscoverContainer>
-        <Sidebar>
-          {this.renderSidebarNav()}
-          {view === 'saved' && (
-            <SavedQueryWrapper>
-              <SavedQueryList organization={organization} savedQuery={savedQuery} />
-            </SavedQueryWrapper>
-          )}
-          {view === 'query' && (
-            <NewQuery
-              organization={organization}
-              queryBuilder={queryBuilder}
-              isFetchingQuery={isFetchingQuery || isLoading}
-              onUpdateField={this.updateField}
-              onRunQuery={this.runQuery}
-              onReset={this.reset}
-              isLoading={isLoading}
-            />
-          )}
-          {isEditingSavedQuery && savedQuery && (
-            <QueryPanel title={t('Edit Query')} onClose={toggleEditMode}>
-              <EditSavedQuery
-                savedQuery={savedQuery}
-                queryBuilder={queryBuilder}
-                isFetchingQuery={isFetchingQuery}
-                onUpdateField={this.updateField}
-                onRunQuery={this.runQuery}
-                onDeleteQuery={this.deleteSavedQuery}
-                onSaveQuery={this.updateSavedQuery}
-                isLoading={isLoading}
-              />
-            </QueryPanel>
-          )}
-          <Feature
-            features={['organizations:discover-basic']}
-            organization={organization}
-          >
-            <SwitchLink
-              href={getDiscoverLandingUrl(organization)}
-              onClick={this.onGoLegacyDiscover}
-            >
-              {t('Go to New Discover')}
-            </SwitchLink>
-          </Feature>
-        </Sidebar>
-
-        <DiscoverGlobalSelectionHeader
+      <React.Fragment>
+        <GlobalSelectionHeader
           organization={organization}
-          projects={currentQuery.projects}
```

Comment:
This gets overwritten by https://github.com/getsentry/sentry/blob/5743740b65d4d26a656cbb0f3481494ceead826d/src/sentry/static/sentry/app/utils/withProjectsSpecified.tsx#L49